src/methods.jl

```diff
@@ -38,3 +38,38 @@ julia> bias_subtraction(frame, bias)
 * [`bias_subtraction!`](@ref)
 """
 bias_subtraction(frame::AbstractArray, bias_frame::AbstractArray) = bias_subtraction!(deepcopy(frame), bias_frame)
+
+
+"""
+    overscan_subtraction(frame::AbstractArray, overscan_frame::AbstractArray; overscan_axis = 2)
+
+In place version of [`overscan_subtraction`](@ref)
+"""
+function overscan_subtraction!(frame::AbstractArray, overscan_frame::AbstractArray; overscan_axis = 2)
```

Comment:
overscan is not a separate frame, it is a region inside of an image. The method should have signature `overscan_subtraction!(frame, idxs; dims=Colon())` and you can do this-

```julia
overscan_region = @view frame[idxs...]
overscan_value = median(overscan dims=dims)
return frame .-= overscan_value
```
```